data/tensorflow/operator/tf_recordio_dataset_test.py

```diff
@@ -2,6 +2,10 @@
 import tensorflow as tf
 from tf_recordio_dataset import RecordIODataset
 
+# Test using CPU 
+import os
+os.environ["CUDA_VISIBLE_DEVICES"]="-1"
```

Comment:
No need to add CUDA_VISIBLE_DEVICES. 
User can use env variable to set it if needed.

Reply:
fixed.